modules/appeals_api/app/controllers/appeals_api/supplemental_claims/v0/supplemental_claims/evidence_submissions_controller.rb

```diff
@@ -1,12 +1,64 @@
 # frozen_string_literal: true
 
 module AppealsApi::SupplementalClaims::V0::SupplementalClaims
-  # rubocop:disable Layout/LineLength
-  class EvidenceSubmissionsController < AppealsApi::V2::DecisionReviews::SupplementalClaims::EvidenceSubmissionsController
-    # rubocop:enable Layout/LineLength
+  class EvidenceSubmissionsController < AppealsApi::ApplicationController
+    include AppealsApi::CharacterUtilities
+    include AppealsApi::JsonFormatValidation
     include AppealsApi::OpenidAuth
+    include AppealsApi::Schemas
+    include AppealsApi::StatusSimulation
+    include SentryLogging
+
+    class EvidenceSubmissionRequestValidatorError < StandardError; end
+
+    skip_before_action :authenticate
```

Comment:
This `authenticate` method is skipped because it comes from the base `AppealsApi::ApplicationController` - our auth is handled by `AppealsApi::OpenidAuth` instead